### PR TITLE
Detailed view for reference data

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by utas-tajona** [comment]
I think overally looks great. it would be good to have input from Antonia and Lizzi about the flow.

if they are ok with adding an extra step to reach the edit page or having a edit button from the grid.  

Also Is there another PR for the backend ? because the API call will have to change for the list view

File: ui/src/App.js [hunk 1]
```diff
@@ -95,6 +96,7 @@ export default function App()  {
               <Route path="/list-file" component={FileList} />
               <Route path="/login" component={Login} />
               <Route path="/form/:entityName/:id?" component={GenericForm} />
+              <Route path="/detailed/:entityName/:id?" component={GenericDetailsView} />
```
[the remark this answers is not in the document]
Comment (pmbohm) [reply ui/src/App.js:99]: At the moment i'm thinking we can use these generically, not just for reference. If it turns out to be unworkable in practice, we can make the distinction.

**Comment by pmbohm** [comment]
_"extra step to reach the edit page"_
@utas-tajona There is the overrides in ['const nonGenericEntities = {'](https://github.com/aodn/nrmn-application/pull/97/files#diff-b464ed0d07d373e8d0e3f0347bea8b8cff81fdf34efa439572cfa42467fa1d4cR82) that will allow us to skip the list step, or we could go to edits by default

File: ui/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ccmoloney) [reply ui/src/App.js]: I think having edit/view in the URL would be good - as suggested by Tanjona

**Pull request merged**